### ISSUE TITLE
Removes own dependencies

### ISSUE THREAD

**Issue by rippedspine** [JamesKyburz/install-missing#68]
When running app with nodemon and using nodemon events to run install-missing on restart, install-missing will remove own dependencies if `require` is removed.

### setup

package.json
```javascript
{
  "start": "nodemon index.js"
}
```

nodemon.json
```javascript
{
  "events": {
    "restart": "./node_modules/.bin/install-missing"
  }
}
```

index.js - remove through2 require and install-missing will stop working
```javascript
var through = require('through2')
```

I might be using this package in a cray-cray wrong way, but thought I'd post an issue anyway ;)

**Comment by JamesKyburz** [comment]
Not a crazy use at all :) The error is caused by how the way npm uninstall works.

I don't use this module much any more and use [dependency-sync](https://npm.im/dependency-sync) instead. That said I added the uninstall functionality from dependency-sync!

Fixed in version `1.1.50`

**Comment by rippedspine** [comment]
~~yass, tried dependency-sync again since that is what I initially wanted to use, must have bodged it up the first time. works like a charm now!~~

Ok... so update. Dependency-sync removes all dependencies :/ I do some requires in runtime, so that might be it? Will post issue on dependency-sync repo.
